test/spec-test4.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env mocha */
+'use strict'
+
+const { expect } = require('aegir/utils/chai')
+const multibase = require('../src')
+const constants = require('../src/constants.js')
+const input = '\x00yes mani !'
+const encoded = [
+  ['identity', '\u0000\x00yes mani !'],
```

Comment:
@hugomrdias is the mixing of `\u0000` and `\x00` normal?

Reply:
It's the prefix